foundations/the_front_end/html_css_basics.md

```diff
@@ -42,9 +42,9 @@ This section contains helpful links to other content. It isn't required, so cons
 * You might find it helpful to bookmark the [Complete Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and the [Complete Guide to Grid](https://css-tricks.com/snippets/css/complete-guide-grid/) from CSS Tricks. These are both excellent references that you'll find handy as you start to apply CSS in projects.
 * Another good page to bookmark for later reference is [Centering in CSS: A Complete Guide](https://css-tricks.com/centering-css-complete-guide/), also from CSS Tricks.
 * [W3 Schools' CSS tutorial](https://www.w3schools.com/css/)
-* [OverAPI's HTML Cheat Sheet](https://overapi.com/html)
-* [OverAPI's CSS Cheat Sheet](https://overapi.com/css)
+* [OverAPI's CSS Cheat Sheet](https://overapi.com/css) and [OverAPI's HTML Cheat Sheet](https://overapi.com/html)
 * [CSSreference.io](https://cssreference.io/) and [HTMLreference.io](https://htmlreference.io)
 * [Learn CSS Layout](https://learnlayout.com/)
 * [CSS Flexbox](https://flexbox.io/) and [CSS Grid](https://cssgrid.io/)
     * These [CSS FlexBox](https://youtu.be/FTlczfR82mQ) and [CSS Grid](https://youtu.be/EFafSYg-PkI) videos by [Dev Ed](https://www.youtube.com/channel/UClb90NQQcskPUGDIXsQEz5Q) might be helpful for you as well!
```

Comment:
Line 50- Remove the indent please! It ends up making it nested. l think it could be good to add the flexbox froggy and grid garden bullet point to the top of the additional resources list.

Reply:
For the indents on the last two list items, the indent was previously there on the page itself so I was following that indent (I was assuming it was originally there to keep the flex/grid items grouped together). Should I still remove the indent and rearrange the froggy/grid items to the top of that list?

Reply:
I did notice that as well at first. But then also noticed that the first bullet the the Additional resources section was also a Flex/Grid thing. Its not organized great. How about keeping the indents so it is nested under the " CSS Flexbox and CSS Grid" section and adding the first bulled (line 42) into that nested list. That way all flex/grid stuff is together.